testing/functional_tests/test_cascaded_architecture_tutorial.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import os
+import pytest
+from pytest_console_scripts import script_runner
+from testing.functional_tests.t_utils import __data_testing_dir__
+from testing.common_testing_util import download_dataset
+
+logger = logging.getLogger(__name__)
+
+@pytest.mark.script_launch_mode('subprocess')
+def test_cascaded_architecture_tutorial(script_runner):
+    download_dataset("data_example_spinegeneric")
+    file_config = os.path.join(__data_testing_dir__, 'cascaded_architecture_tutorial_config.json')
```

Comment:
note for this to work you have to upload the config.json file to the data_functional_test directory just like automated_test_config... I thought about changing the config inside the test but I do not think it is a good idea since you would be assuming other fields in the config.json which is subject to change